api/sys_monitor.go

```diff
@@ -18,6 +20,12 @@ func (c *Sys) Monitor(ctx context.Context, logLevel string) (chan string, error)
 		r.Params.Add("log_level", logLevel)
 	}
 
+	if logFormat == "" || logFormat == logging.UnspecifiedFormat.String() {
```

Comment:
Why check UnspecifiedFormat?

Reply:
Some of our tests rely upon parsing of the logformat environment variable which if unset is set to "unspecified".